pkg/dao/dao.go

```diff
@@ -2,36 +2,23 @@ package dao
 
 import (
 	"database/sql"
-	"errors"
-
-	"github.com/aerogear/aerogear-metrics-api/pkg/mobile"
 )
 
 type MetricsDAO struct {
 	db *sql.DB
 }
 
 // Create a metrics record
-func (m *MetricsDAO) Create(metric mobile.Metric) (mobile.Metric, error) {
-	return metric, errors.New("Not Implemented yet")
-}
-
-// Update an existing job
-// Not sure if we need this
-func (m *MetricsDAO) Update() {
-
+func (m *MetricsDAO) Create(clientId string, metricsData []byte) error {
+	_, err := m.db.Exec("INSERT INTO mobileappmetrics(clientId, data) VALUES($1, $2)", clientId, metricsData)
```

Comment:
[Optional] mobileappmetrics table is used in various SQL calls. DAO here works fine, but maybe it can also handle the actual table creation logic (to fully extract that there is some SQL/postgre involved) Alternative will be to just use constant for `mobileappmetrics` table name. It's not big deal, since we already have abstraction for SQL queries, but creation of the table seems to be leaking that we using sql.db interface.

Reply:
+1 I think a constant/config item for the database table is the best option